lib/dynflow.rb

```diff
@@ -49,4 +49,16 @@ class Error < StandardError
   require 'dynflow/semaphores'
   require 'dynflow/throttle_limiter'
   require 'dynflow/config'
+
+  if defined? ::ActiveJob
+    require 'dynflow/active_job/queue_adapter'
+
+    class Railtie < Rails::Railtie
```

Comment:
I could not find any example of ActiveJob usage without Rails - so I don't think they support it to work without it, it's even in the same repository as Rails (rails/rails). The reason why I check for `::ActiveJob` is in case someone is using Dynflow with some Rails version that does not have ActiveJob.